### DISCUSSION TITLE
-r   The more, the better? Or are two the best

### DISCUSSION THREAD

**Discussion by Rempage** [Chia-Network/chia-blockchain#2257] [Q&A]
-r   The more, the better? Or are two the best

**Comment by farmingtonschia** [comment]
2-4 threads is  recommended per notes on chia.net. On my 4th gen i7 setting it -r 4 and -b 6750 produced the best metrics

**Reply by Rempage** [reply]
See the official saying -b 3390 has been optimized

**Reply by farmingtonschia** [reply]
thank you - i missed that note on optimal -b; i will try with -b 3390; meanwhile as regards the -r stats for phase 1 timing -r 2 = 8810 s; -r4 avg 7300;

**Reply by farmingtonschia** [reply]
please share stats and cpu if possible

**Reply by Rempage** [reply]
cpu amd r7  3800xt.   
memory   32g
Last night, we tested 8 tasks at the same time, - R 2 was slower than - R 16. The overall time difference is about 1800 seconds

Now testing - R 4
